sunpy/map/header_helper.py

```diff
@@ -119,6 +131,15 @@ def make_fitswcs_header(data, coordinate, reference_pixel: u.pix = None,
     meta_wcs['cdelt1'], meta_wcs['cdelt2'] = (scale[0]*meta_wcs['cunit1']/u.pixel,
                                               scale[1]*meta_wcs['cunit2']/u.pixel)
 
+    if rotation_angle is not None:
+        meta_wcs['crota2'] = rotation_angle.to_value()
+
+    if rotation_matrix_pc is not None:
```

Comment:
We should check that only `rotation_angle` or `rotation_matrix` is set. We should also validate that the rotation matrix is the right shape.

Reply:
silly question - what do you meet by 'is set'?

Also yes will add in that check

Reply:
I mean "is not None" in this case, i.e. just that the user hasn't specified both.